packages/create-sitecore-jss/src/templates/vue/package.json

```
@@ -0,0 +1,90 @@
+{
+  "name": "jss-sample-vue",
+  "version": "20.0.0-canary.77",
```

```suggestion
  "version": "20.0.0-canary",
```
Can remove the specific canary version, same with the sitecore-jss dependencies below.